migrations/sql/R__stored_procedure_for_permit_and_permitee_information.sql

```diff
@@ -576,77 +578,149 @@ CREATE OR REPLACE FUNCTION transfer_premit_permitee_information() RETURNS void A
                     permit_status_code  ,
                     create_user         ,
                     create_timestamp    ,
                     update_user         ,
-                    update_timestamp    ,
-                    authorization_end_date
+                    update_timestamp    
                 )
                 SELECT
                     new_permit.permit_guid         ,
                     new_permit.mine_guid           ,
                     new_permit.permit_no           ,
-                    new_permit.received_date       ,
-                    new_permit.issue_date          ,
                     new_permit.permit_status_code  ,
                     'mms_migration'                ,
                     now()                          ,
                     'mms_migration'                ,
-                    now()                          ,
-                    new_permit.authorization_end_date
+                    now()                          
                 FROM new_permit
                 INNER JOIN ETL_MINE ON
                     new_permit.mine_guid = ETL_MINE.mine_guid
                 RETURNING 1
             )
             SELECT COUNT(*) FROM inserted_rows INTO insert_row;
+			
+            RAISE NOTICE '.. Update ETL_PERMIT and all_permit_info with newly inserted permit_guids for new amendments';
+			UPDATE ETL_PERMIT SET permit_guid = (select permit_guid from permit WHERE ETL_PERMIT.permit_no=permit.permit_no and ETL_PERMIT.mine_guid = permit.mine_guid limit 1)
+			where permit_amendment_guid NOT IN (
+				SELECT permit_amendment_guid
+				FROM permit_amendment
+				);	
+				
+				
+            RAISE NOTICE '.. Insert new MMS ETL_PERMIT records into permit amendment';
+            WITH
+            --Select only new entry in ETL_PERMIT table
+            new_permit_amendment AS (
+                SELECT *
+                FROM ETL_PERMIT
+                WHERE permit_amendment_guid NOT IN (
+                    SELECT permit_amendment_guid
+                    FROM permit_amendment
+                )
+            ), inserted_rows AS (
+                INSERT INTO permit_amendment (
+                    permit_id              			,
+                    received_date          			,
+                    issue_date             			,
+                    authorization_end_date 			,
+                    permit_amendment_type_code      ,
+					permit_amendment_status_code    ,
+                    create_user            			,
+                    create_timestamp       			,
+                    update_user            			,
+                    update_timestamp       			
+                )
+                SELECT
+                    permit.permit_id				         	,
+                    new_permit_amendment.received_date       	,
+                    new_permit_amendment.issue_date          	,
+                    new_permit_amendment.authorization_end_date	,
+                    'AMD'									  	,
```

Comment:
I won't block a PR for crazy comma alignment, but I will point it out

Reply:
Also not sure why it's showing up on github like this but locally in both VSCode and Notepad++ it appears as expected with all commas lined up :)